permit_get/__init__.py

```diff
@@ -52,12 +52,18 @@ def main(req: func.HttpRequest) -> func.HttpResponse:
                     "APPLICATION_CREATION_DATE",
                     "Applicant_name",
                     "role"]
+                max_show = 5
```

Comment:
let's make a limit parameter from the endpoint or at least an ENV var if we want to set this globally. taking a limit parameter probably would be more flexible.

Reply:
https://github.com/SFDigitalServices/pts-microservice-fn-py/pull/5/commits/568c95f1d181af4b4d5e1a0880b702a6c2e00104

https://github.com/SFDigitalServices/pts-microservice-fn-py/pull/5/commits/8f67198de5cf3cc33b460dc5dd4be51221b94ccb